handlers/graph.go

```diff
@@ -181,20 +184,50 @@ func buildNamespaceTrafficMap(namespace string, o options.Options, client *prome
 	// fetch the internally originating request traffic time-series
 	intVector := promQuery(query, time.Unix(o.QueryTime, 0), client.API())
 
+	// 4) query for TCP  traffic. Analogously to HTTP, three queries:
+	//   - first, traffic originating from "unknown"
+	//   - second, traffic originating from a workload outside of the namespace
+	//   - lastly, traffic originating from a workload inside of the namespace
+	groupByTcp := "source_workload_namespace,source_workload,source_app,source_version,destination_workload_namespace,destination_service_name,destination_workload,destination_app,destination_version"
+	query = fmt.Sprintf("sum(rate(%v{reporter=\"destination\",source_workload=\"unknown\",destination_workload_namespace=\"%v\"} [%vs])) by (%v)",
+		tcpMetric,
```

Comment:
Thinking about it I wonder whether "received" would be more interesting than "sent" because received payload tends to have more size and more variance.  I'm ok if we leave it as sent for now.  Also, we may find that people want both.

Reply:
The three queries make sense to me, completely analogous to the http queries.

Reply:
I leave this as is for now. Perhaps when we agree what (and how) to show in the edges as labels, we can change the TS to query.